src/test/fuzz/netbase_dns_lookup.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <netaddress.h>
+#include <netbase.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+namespace {
+FuzzedDataProvider* fuzzed_data_provider_ptr = nullptr;
```

Comment:
style-nit: Any reason for this namespace and pointer? Looks like a lambda should be able to do the same in less code and smaller scope

Reply:
Using a lambda here is a good idea: less code, smaller scope and overall beautiful :)

TBH I don't remember why I didn't do it that way nine months ago when this code was written: a lambda is strictly better AFAICT :)